pkgs/development/libraries/libax25/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, glibc
+, static ? stdenv.hostPlatform.isStatic
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libax25";
+  version = "0.0.12-rc5";
+
+  buildInputs = [ glibc ] ++ lib.optional static [ glibc.static ];
+
+  # Due to recent unsolvable administrative domain problems with linux-ax25.org,
+  # the new domain is linux-ax25.in-berlin.de
+  src = fetchTarball {
+    url = "https://linux-ax25.in-berlin.de/pub/ax25-lib/libax25-${version}.tar.gz";
+    sha256 = "1xnxab6n8w8b00dmg5idb6xwqs3llqsl0wv6s3xkf72pn1p1f7k5";
+  };
+
+  configureFlags = [ "--sysconfdir=/etc" ];
+
+  LDFLAGS = lib.optionalString static "-static-libgcc -static";
+
+  meta = with lib; {
+    description = "A set of functions making it easier to write hamradio programs";
```

Comment:
The summary from `libax25.specfile.in` is "AX.25 library for hamradio applications" - should we make this match?

Reply:
Good catch, updated!